doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -464,6 +470,92 @@ disabled, the liveness check uses `GET localhost:8080/ping`
 *  `vertx.health.path` set to "/ping" and `vertx.health.readiness.path` set to "/ready"  => the readiness check uses
 `GET localhost:8080/ready` , the liveness check uses the `GET localhost:8080/ping`
 
+[[f8-maven-scm-enricher]]
+==== f8-maven-scm-enricher
+
+This enricher adds additional SCM related metadata to deployments and services.  These metadata will be added only if
```

Comment:
Its added to more than only deployment and services. Maybe say 'all objects supporting annotations' (which are effectively every object) ?